Tests/Honeycomb/ReadableSpanMock.swift

```diff
@@ -0,0 +1,78 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+import Foundation
+import OpenTelemetryApi
+
+@testable import OpenTelemetrySdk
+
+class ReadableSpanMock: ReadableSpan {
+    var hasEnded: Bool = false
+    var latency: TimeInterval = 0
+
+    var kind: SpanKind {
+        return .client
+    }
```

Comment:
can this be written as

`var kind: SpanKind { .client }`

Reply:
appears so, will change